src/MatrixDotNet/Extensions/Statistics/Minimum.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Collections;
+using System.Numerics;
```

Comment:
is it needed?

Reply:
Yes, I think to enhance the transpose matrix with happening Numerics library